src/components/RadioGroup/RadioGroup.jsx

```diff
@@ -0,0 +1,166 @@
+   * Both a title and description can be included in the legend.
+   */
+  legend: PropTypes.shape({
+    title: PropTypes.node.isRequired,
+    description: PropTypes.node,
+  }),
+  /**
+   * Additional classes to add
+   */
+  className: PropTypes.string,
+  /**
+   * Mark the radio group as invalid and display a validation message.
+   * Pass a string or node to render a validation message below the input.
+   */
+  error: PropTypes.oneOfType([
+    PropTypes.bool,
+    PropTypes.string,
+    PropTypes.node,
+  ]),
+  /**
+   * If the radio group should be disabled and not focusable
+   */
+  isDisabled: PropTypes.bool,
+  /**
+   * Determines if radio group is required or not. (Label will have an asterisk if required)
+   */
+  isRequired: PropTypes.bool,
+  /**
+   * Callback function to call on focus event.
+   */
+  onFocus: PropTypes.func,
+  /**
+   * Callback function to call on blur event.
+   */
+  onBlur: PropTypes.func,
+  /**
+   * The value of selected radio input
+   */
+  selectedOption: PropTypes.string,
+};
+
+const defaultProps = {
+  legend: undefined,
+  className: '',
+  error: false,
+  isDisabled: false,
+  isRequired: false,
+  onFocus: undefined,
+  onBlur: undefined,
+  selectedOption: undefined,
+};
+
+const RadioGroup = ({
+  legend,
+  name,
+  className,
+  error,
+  isDisabled,
+  isRequired,
+  onChange,
+  onFocus,
+  onBlur,
+  options,
+  selectedOption,
+}) => {
+  const handleFocus = e => {
+    if (onFocus) onFocus(e);
+  };
+
+  const handleBlur = e => {
+    if (onBlur) onBlur(e);
+  };
+
+  const groupClasses = classNames(
+    'radioGroup',
+    className,
+    { error },
+  );
+
+  const legendClasses = classNames(
+    'legend',
+    { error },
+  );
+
+  return (
+    <div className={classNames('Palmetto-RadioGroup', groupClasses)}>
+      <fieldset className="fieldset">
+        {legend && (
+          <legend className={legendClasses}>
+            {legend.title}
+            {isRequired && <span className="font-size-sm">&nbsp;*</span>}
+            {legend.description && <div className="description">{legend.description}</div>}
+          </legend>
+        )}
+        <div className="options">
+          {options.map(option => {
+            const labelProps = {
+              inputId: option.id,
+              labelText: option.label,
+              isDisabled: isDisabled || option.disabled,
+              displayInline: true,
+              hasError: !!error,
+            };
+
+            return (
+              <div className="option" key={option.id}>
+                <input
+                  id={option.id}
+                  type="radio"
+                  name={name}
+                  className="radioInput"
+                  value={option.value}
+                  checked={selectedOption === option.value}
+                  onChange={onChange}
+                  onFocus={handleFocus}
+                  onBlur={handleBlur}
+                  disabled={isDisabled || option.disabled}
```

Comment:
@JCarmine should this be `option.isDisabled` I believe that is the shape noted in the proptypes, unless I am missing something.

Reply:
Seems that because that specific part of the shape is not required, it is not throwing a proptypes error, but there is a mismatch.